src/channel.cpp

```diff
@@ -611,8 +614,8 @@ EPutDataStat CChannel::PutAudioData ( const CVector<uint8_t>& vecbyData,
                     eRet = PS_AUDIO_ERR;
                 }
 
-                // manage audio fade-in counter
-                if ( iFadeInCnt < iFadeInCntMax )
+                // manage audio fade-in counter, after channel is identified
+                if ( bIsIdentified && iFadeInCnt < iFadeInCntMax )
```

Comment:
Turning the boolean evaluation order around should in theory save one instruction in the most common path?

```suggestion
                if ( iFadeInCnt < iFadeInCntMax && bIsIdentified )
```

- Whether fade in is still active has to be checked in all cases (as before)
- Whether the client is identified will only need to be checked if fade in is active